cdap-master/src/main/java/co/cask/cdap/data/tools/ReplicationStatusTool.java

```diff
@@ -67,13 +77,25 @@
   private static Long masterShutdownTime = 0L;
   private static String inputStatusFileName = null;
   private static String outputStatusFileName = null;
+  private static String dirsFileName = null;
   private static boolean debugDump = false;
-  private static String delimiter = ":";
+  private static String delimiter = "#";
+  private static String checksumsEnd = "EndofChecksums";
+  private static String tablesEnd = "EndofTables";
+
+  // Default paths to be pulled from configuration
+  private static final String[] ALL_CHECKSUM_PATHS_DEFAULT = {
```

Comment:
From the last discussion, the plan was to use only the snapshot files for hdfs checksum, but could be that more files could be added later on. I have kept the code in the tool scalable to other files. The user can use -f option to provide a file with multiple hdfs directories to be scanned.